code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -532,7 +531,6 @@
 	attack_sound = 'sound/weapons/slice.ogg'
 	miss_sound = 'sound/weapons/slashmiss.ogg'
 	sharp = 1
```

Comment:
So wait, Unathi, Tajaran, Vulp, Vox, and Kidan can also dismember people with bare hands?